source/rules/rxjsSuffixSubjectsRule.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/rxjs-tslint-rules
+ */
+/* tslint:disable:no-use-before-declare */
+
+import { tsquery } from "@phenomnomnominal/tsquery";
+import * as Lint from "tslint";
+import * as ts from "typescript";
+import { couldBeType } from "../support/util";
+
+const defaultTypesRegExp = /^EventEmitter$/;
+
+export class Rule extends Lint.Rules.TypedRule {
+  public static metadata: Lint.IRuleMetadata = {
+    description:
+      "Ensures subjects are suffixed with 'Subject'.",
+    options: {
+      properties: {
+        functions: { type: "boolean" },
+        methods: { type: "boolean" },
```

Comment:
<s>`functions` and `methods` don't appear to be used - at least not in the test that I read before I read the code. If they're not used, they should be removed from the options.</s>

Never mind. A closer look at the code shows that they are used.